src/admin/config.yml

```diff
@@ -0,0 +1,59 @@
+backend:
+  name: git-gateway
+  branch: dev
```

Comment:
As we talked yesterday in the channel, the plan is to use different config files for `dev` and `main` branches. Do you have thoughts about how to handle this difference during the PR merge process.

Reply:
My understanding of git merging is that if we:

1. Merge this to dev.
2. Merge dev to main.
3. Update main's config.yml to point to main instead of dev.

Future merges from dev won't update that line since it will have been modified more recently in main than in dev. We'll need to verify this behaviour though.